lib/hydra/works/services/virus_checker_service.rb

```diff
@@ -30,8 +30,9 @@ def local_path_for_file(file)
         return file.path if file.respond_to?(:path)
         Tempfile.open('') do |t|
           t.binmode
-          t.write(file.content.read)
-          file.content.rewind
```

Comment:
Does removing this change the state of the file at the end of virus checking?  Or do we not need to do that any more because streaming the file doesn't consume it the same way calling `content` did?